### ISSUE TITLE
feature request: `extraContext:editorCursorPreviousChar`

### ISSUE THREAD

**Issue by scarf005** [rioj7/extra-context#1]
## Summary

editorCursorPreviousChar will make it easier to use https://github.com/microsoft/vscode/pull/160270 .

## Details

```json
  {
    "key": "backspace",
    "command": "editor.action.removeBrackets",
    "when": "editorTextFocus && extraContext:editorCursorPreviousChar == '}'"
  },
```

**Comment by rioj7** [comment]
@scarf005 

fixed in 15445b9

Can you try v0.6.0

It also returns `}` if it is inside a string.

**Comment by scarf005** [comment]
thanks!
